main/Pipfile

```diff
@@ -4,6 +4,7 @@ url = "https://pypi.org/simple"
 verify_ssl = true
 
 [dev-packages]
+pytest = "*"
 pytest-cov = "*"
 
```

Comment:
Can you set the python version to 3.10?

On my machine, it is not working with python 3.7.
Python 3.10, which is specified in the build workflow, is working.